CONTRIBUTING.md

```
@@ -1,3 +1,3 @@
-- Write your changes to `index.js` and not `dist/index.js`
-- Make sure your changes are [prettified](https://prettier.io/)
-- Run `yarn build` before commiting your changes
+- Write your changes to `index.mjs` and not `dist/index.mjs`
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)